docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -18,7 +18,38 @@ Following these steps, you can implement a typical OpenVINO™ Runtime inference
 
 ![ie_api_use_cpp]
 
-## Step 1. Create OpenVINO™ Runtime Core 
+## Step 1. Create Cmake Script
+
+This step may differ for different projects. In this example, a C++ & C application is used, together with CMake for project configuration.
+`
+Create a structure for the project:
+project/
+   ├── CMakeLists.txt  - CMake file to build
+   ├── ...             - Additional folders like includes/
+   └── src/            - source folder
+       └── main.c      - [Optional] For C sample
+       └── main.cpp    - [Optional] For C++ sample
```

Comment:
Why? It is unreal structure, honestly I prefer tabs, as it was before, my ask was just to add some sentence to differentiate  cmake script and project structure

Reply:
misunderstand, have split the "project structure" and "cmake script". Pls help review.